pynamodb/models.pyi

```diff
@@ -94,7 +94,16 @@ class Model(metaclass=MetaModel):
     @classmethod
     def describe_table(cls): ...
     @classmethod
-    def create_table(cls: Type[_T], wait: bool = ..., read_capacity_units: Optional[Any] = ..., write_capacity_units: Optional[Any] = ...): ...
+    def create_table(
+        cls: Type[_T],
+        wait: bool = ...,
+        read_capacity_units: Optional[Any] = ...,
+        write_capacity_units: Optional[Any] = ...,
+        billing_mode: Optional[Any] = ...,
+        ignore_update_ttl_errors: Optional[bool] = ...
```

Comment:
not just `: bool`?

Reply:
because you can't have required params after optional ones

Reply:
You're thinking of default params. "Optional" just means that it can be None.

Reply:
You're right - I'll change